src/main/java/org/opentripplanner/transit/service/TransitModel.java

```diff
@@ -55,7 +55,26 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * Repository for Transit entities.
+ * The TransitModel groups together all instances making up OTP's primary internal representation
+ * of the public transportation network. Although the names of many entities are derived from
+ * GTFS concepts, these are actually independent of the data source from which they are loaded.
+ * Both GTFS and NeTEx entities are mapped to these same internal OTP entities.
```

Comment:
You could mention the following policy: if a concept exists in both GTFS and Netex, we will use the GTFS name for the internal model. If not, then the Netex one is used.